dojo/tools/nmap/parser.py

```diff
@@ -96,9 +96,12 @@ def get_findings(self, file, test):
                         service_info += (
                             "**Extra Info:** {}\n".format(port_element.find("service").attrib["extrainfo"])
                         )
-
                     description += service_info
-
+                if port_element.find("script") is not None:
+                    if "id" in port_element.find("script").attrib:
+                        description += "**Script ID:** " + port_element.find("script").attrib["id"] + "\n"
+                    if "output" in port_element.find("script").attrib:
+                        description += "**Script Output:** " + port_element.find("script").attrib["output"] + "\n"
```

Comment:
What do you think about the following? Increase readability for me.
```suggestion
                if script := port_element.find("script"):
                    if script_id := script.attrib.get("id"):
                        description += f"**Script ID:** {script_id}\n"
                    if script_output in port_element.find("script").attrib.get("output"):
                        description += f"**Script Output:** {script_output}\n"
```

Reply:
And decrease the number of `.find(...)` calls